### PR TITLE
feat: gcp support

### PR DESCRIPTION
**Pull request by abalam666**
## Changes

Starting GCP support

### CODE REVIEW

**Comment by loopingz** [comment]
The linter and prettier are in the root of the repository

File: packages/gcp/src/services/storage.ts [hunk 1]
```diff
@@ -0,0 +1,661 @@
...
+  endpoint?: string;
+  prefix?: string;
+  CloudFormation: any;
+  CloudFormationSkip: boolean = false;
+
+  /**
+   * Bucket
+   */
+  private _bucket?: string;
+  get bucket(): string {
```
Comment (loopingz) [inline packages/gcp/src/services/storage.ts:56]: the loader of parameters does `Object.assign(this, params)`, I do not think we want to have the webda.config.json to have a parameter `_bucket`

Comment (abalam666) [reply packages/gcp/src/services/storage.ts]: removed

**Comment by abalam666** [comment]
> The linter and prettier are in the root of the repository

i see the prettier but the linter... ?

**Comment by loopingz** [comment]
You are right I though prettier was enough, what eslint brings to the table?

**Comment by loopingz** [comment]
Any updates?

**Comment by loopingz** [comment]
Closed has it is superseded by #162

**Pull request closed without merging**